src/main/java/org/benetech/servicenet/util/UrlNormalizationUtils.java

```diff
@@ -27,7 +28,16 @@ public static String normalize(String url) {
         if (result.toUpperCase(Locale.ROOT).endsWith(TRAILING)) {
             result = result.substring(0, result.length() - 1);
         }
+        for (String domen:DOMENS) {
+            if ((result.toUpperCase(Locale.ROOT).contains(domen))) {
+                return result;
+            }
+        }
+        if (result.toUpperCase(Locale.ROOT).contains(TRAILING)) {
```

Comment:
Should not this part be before domain check?

Reply:
I think not, because for those few domains you want to return the exact url

Reply:
Yes, as @pmuchowski wrote

Reply:
ok, cool